src/main/resources/templates/email/reciept.hbs

```diff
@@ -0,0 +1,47 @@
+<html>
+
+<head>
+    <title>reciept</title>
+
+    <style>
+        .tr {
+            background : grey;
+
+        }
+        .td {
+            min-width : 30%;
+        }
+
+
+   </style>
+</head>
+
+<body style="border-collapse: collapse;">
+    <div style="border: 1px #eee solid;">
+        <div style="background : #eee; padding: 0 10px;">
+            <p style="margin : 0px;">Order <b>{{order.id}}</b> from th_e <b>{{order.date}}</b></p>
+            <p style="margin : 0px;">{{user.firstname}} {{user.lastname}}, {{user.street}} {{user.housenumber}} {{user.zipcode}}</p>
+        </div>
+
+
+        <table style="margin: 0 auto;">
+            {{#each order.orderItems}}
+                <tr style="#eee;">
```

Comment:
Does not appear to be valid css

Reply:
Was removed.
Please close this conversation if you are fine with it @NPapsdorf .